docs/hadoop.adoc

```diff
@@ -1,20 +1,70 @@
 [[hadoop-tp3]]
-== JanusGraph with TinkerPop's Hadoop-Gremlin
+== JanusGraph with Tinker Pop's Hadoop-Gremlin
 
-JanusGraph-Hadoop works with TinkerPop's hadoop-gremlin package for
+JanusGraph-Hadoop works with Tinker Pop's hadoop-gremlin package for
 general-purpose OLAP.
```

Comment:
Add to this introduction:

This chapter describes how to leverage [Apache Hadoop](https://hadoop.apache.org) and [Apache Spark](https://spark.apache.org) to configure JanusGraph for distributed graph processing. These steps will provide an overview on how to get started with those projects, but please refer to those project communities to become more deeply familiar with them.